KNOWN_ISSUES.md

```diff
@@ -19,7 +19,7 @@
   Resolved in 4.7.0-0.okd-2021-06-13-090745 - latest stable FCOS can be used to create a cluster.
 
 ## [Invalid signature error when attempting cluster updates](https://github.com/openshift/okd/issues/605)
-  **Effected Versions:** 4.7.0-0.okd-2021-04-11-124433 and 4.7.0-0.okd-2021-05-22-050008
+  **Affected Versions:** 4.7.0-0.okd-2021-04-11-124433 and 4.7.0-0.okd-2021-05-22-050008
```

Comment:
Lets keep this change outside of this PR

Reply:
Okay, I just forced pushed the single commit to my fork with only this section, I'll drop the copy edits in another PR.